### DISCUSSION TITLE
/

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#2101] [Discussão]
/

**Comment by wilsonneto-swe** [comment]
Dele eu não fiz, mas se ajudar, separei uma lista dos melhores cursos de desenvolvimento JS (Node e react) que já fiz, eu faço vários até hoje, mesmo com 10 anos de carreira, sempre da para aprender algo novo ou pegar algum insight.

Cursos que indico demais:
[https://github.com/wilsonneto-dev/Melhores_Cursos_Online](https://github.com/wilsonneto-dev/Melhores_Cursos_Online)

**Reply by igorlvicente** [reply]
>eu faço vários até hoje, mesmo com 10 anos de carreira, sempre da para aprender algo novo ou pegar algum insight

Parece tão óbvio mas às vezes a gente não pensa nessas coisas. Obrigado pela inspiração, @wilsonneto-dev, vou começar a fazer isso também.

**Comment by felquis** [comment]
Olá @RodrigoVieira19, tudo bem?

É importante vc manter seus tópicos disponíveis para ajudar outras pessoas com duvidas semelhantes no futuro.
